src/afl-forkserver.c

```diff
@@ -1665,6 +1670,14 @@ afl_fsrv_run_target(afl_forkserver_t *fsrv, u32 timeout,
   MEM_BARRIER();
 #endif
 
+  if (fsrv->custom_fuzz_run) {
+
+    fsrv->total_execs++;
```

Comment:
we prefer `++fsrv->custom_fuzz_run` because it is faster. silly I know but for several compilers this is true.

Reply:
Fixed. Thanks!